addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/callbacks/FritzAhaDiscoveryCallback.java

```diff
@@ -29,9 +28,15 @@
  *         DECT
  *
  */
-public class FritzAhaDiscoveryCallback extends FritzAhaReauthCallback {
+public class FritzAhaDiscoveryCallback extends FritzAhaXMLCallback {
+    /**
+     * logger
```

Comment:
I'm no fan of comments which add no value, everyone could have guessed this is a logger :-P